storage-service/src/test/java/io/quarkus/qe/consumers/UpdateRepositoryRequestConsumerTest.java

```diff
@@ -66,7 +68,7 @@
     public void setup() {
```

Comment:
line 60:

Could you remove logMarshaller, seems that is not used. (maybe another PR forgot it!) 
```

@Inject
    LogMarshaller logMarshaller;
```

Reply:
Well spotted! Deleted.